.storybook/config.js

```diff
@@ -1,10 +1,15 @@
-import { configure } from "@storybook/react";
+import { configure, addDecorator } from "@storybook/react";
+import { withKnobs } from "@storybook/addon-knobs";
+import "bootstrap/dist/css/bootstrap.min.css";
 import "@storybook/addon-console";
```

Comment:
Enable knobs globally. Import Bootstrap so it's available for all stories.

Reply:
Completed